### PR TITLE
🚚 Fix netlify preview url

### PR DESCRIPTION
**Pull request by vikingviolinist**

### CODE REVIEW

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -3,7 +3,7 @@
   import '/src/app.css';
   import { Cart, Footer, Header, KeyprezName } from '$lib';
 
-  let contentHidden = !$page.url.origin.match(/localhost|deploy-preview/g);
+  let contentHidden = !$page.url.origin.match(/localhost|keyprez.netlify.app/g);
```
Comment (csandven) [inline src/routes/+layout.svelte:6]: Does this mean that some content is hidden from the site on dev?

**Review by csandven — CHANGES_REQUESTED** [review]
Can we add the url as an env var instead?

**Comment by vikingviolinist** [comment]
> Can we add the url as an env var instead?

You mean as a const in the config file? There're no env vars on frontend.

File: src/routes/+layout.svelte (hunk 1 above)
Comment (vikingviolinist) [reply src/routes/+layout.svelte:6]: No, it was hidden completely, because netlify changed the preview url.

**Review by csandven — APPROVED** [review]

**Pull request merged**